README.md

```diff
@@ -374,10 +377,23 @@ dablin_gtk -d ~/bin/dab2eti -c 11D -C 5C,7B,11A,11C,11D -s 0xd911
 ```
 
 You also can use an Airspy or Airspy mini, but you have to specify the
-gain in the `-C` parameter as the auto gain does not work right here.
+`-G` parameter.
 
 ```sh
-dablin_gtk -D eti-cmdline -d ~/bin/eti-cmdline-airspy -c 11D -C 11D:80
+dablin_gtk -D eti-cmdline -d ~/bin/eti-cmdline-airspy -c 11D -C 11D -G
+
```

Comment:
This empty line is part of the code block, but is not needed as part of it. Please swap this and the following line.